tests/integration/month.rs

```diff
@@ -1,4 +1,6 @@
-use time::Month::*;
+use std::convert::TryFrom;
+
+use time::{Month, Month::*};
```

Comment:
```suggestion
use time::Month::{self, *};
```

Disregard if rustfmt doesn't like this.